urllib3/connection.py

```diff
@@ -195,10 +195,21 @@ class VerifiedHTTPSConnection(HTTPSConnection):
     Based on httplib.HTTPSConnection but wraps the socket with
     SSL certification.
     """
-    cert_reqs = None
     ca_certs = None
     assert_fingerprint = None
 
+    def __init__(self, host, port=None, key_file=None, cert_file=None,
+                 strict=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
+                 ssl_context=None, ssl_version=None, cert_reqs=None, **kw):
+        UnverifiedHTTPSConnection.__init__(self, host, port, key_file,
+                                           cert_file, strict, timeout,
+                                           ssl_context, ssl_version, **kw)
+
+        if ssl_context is None:
```

Comment:
Nit (optional): In general unless `None` has a special meaning, we usually just do `if foo: ...` or `if not foo: ...`